### ISSUE TITLE
Exception in thread "main" java.lang.IllegalArgumentException: Collection is empty

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Valyrox/lanterna#4]
```
Hello, when I try to run your example of GUIScreen (PanelTest)  I get:

[x@dist]$ java -classpath lib/lanterna-1.0.1.jar -jar CDmenuPrototype.jar 
Exception in thread "main" java.lang.IllegalArgumentException: Collection is 
empty
    at java.util.EnumSet.copyOf(EnumSet.java:166)
    at org.lantern.gui.TextGraphics.drawString(Unknown Source)
    at org.lantern.gui.TextGraphics.fillRectangle(Unknown Source)
    at org.lantern.gui.GUIScreen.repaint(Unknown Source)
    at org.lantern.gui.GUIScreen.update(Unknown Source)
    at org.lantern.gui.GUIScreen.doEventLoop(Unknown Source)
    at org.lantern.gui.GUIScreen.showWindow(Unknown Source)
    at atest.PanelTest.main(PanelTest.java:85)


It is the same when I try simple stuff as well.
Thanks
```

Original issue reported on code.google.com by `zham...@gmail.com` on 8 Dec 2011 at 4:02

**Comment by GoogleCodeExporter** [comment]
```
java version "1.6.0_22"
OpenJDK Runtime Environment (IcedTea6 1.10.4) (ArchLinux-6.b22_1.10.4-1-i686)
OpenJDK Server VM (build 19.0-b09, mixed mode)
```

Original comment by `zham...@gmail.com` on 8 Dec 2011 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
Right, I think this one is fixed in the upcoming 1.0.2 release.
```

Original comment by `mab...@gmail.com` on 9 Dec 2011 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
Yes, confirmed. If you checkout from the repository, it will work, or wait a 
day and I'll build the release.
```

Original comment by `mab...@gmail.com` on 9 Dec 2011 at 12:48
- Changed state: **Verified**
